### DISCUSSION TITLE
Welcome to ji-cloud Discussions!

### DISCUSSION THREAD

**Discussion by dakom** [ji-devs/ji-cloud#1400] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mastapegs** [comment]
Hello, World!

I love seeing how this repo mixes together web components and Rust and since this is a public repo, I wanted to sharpen my teeth with some Rust/Lit, and see if there were any issues an outside contributor could maybe take a look at?

Fantastic job with this repo again! Beautiful code <3
